### ISSUE TITLE
deprecation of 3rd party cookies most web browsers pretty soon

### ISSUE THREAD

**Issue by wgwz** [ccnmtl/django-lti-provider#281]
https://github.com/ccnmtl/django-lti-provider/blame/master/README.md#L96-L104

It looks like there will be inconvenient from a user perspective to use signed cookies to store information about the Canvas POST request in the Django session. 3rd party cookies seemed to be be blocked by default in Chrome now. And they will be phased out entirely in a couple years:

https://www.theverge.com/2020/1/14/21064698/google-third-party-cookies-chrome-two-years-privacy-safari-firefox

Server-side sessions should probably be recommended in the README instead, perhaps with an example of using sqlite. Maybe that already exists in the examples? Just wanted to flag it.

**Comment by wgwz** [comment]
BTW I realize I am the one who added these :-) I just wanted to flag the issue to help cleaning it up

**Comment by wgwz** [comment]
This is a critical resource that specifies info about how Instructure plans to resolve these issues:

https://community.canvaslms.com/t5/Developers-Group/SameSite-Cookies-and-Canvas/ba-p/257967

**Comment by wgwz** [comment]
> tl;dr tools that utilize cookies and integrate with Canvas will need to add SameSite=None and Secure attributes to their cookies to maintain current behavior.
I'm going to add this to the README
